.travis.yml

```diff
@@ -1,38 +1,47 @@
-language: php
-
-php:
-  - 7.1
-  - 7.2
-  - 7.3
+dist: xenial
+sudo: true
 
-env:
-  - SYMFONY_VERSION="~2.3.0"
-  - SYMFONY_VERSION="~2.7.0"
-  - SYMFONY_VERSION="~3.2.0"
-
-branches:
-  only:
-    - master
+language: php
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
+matrix:
+  fast_finish: true
+  include:
+    # all supported PHP versions with lowest dependencies
+    - php: 7.1
+      env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest"
+# @TODO: uncomment when cassandra extension will support >= php7.2
```

Comment:
Is it clear in the README? 🤔

Reply:
we display this note:

> You need to [install the offical datastax php driver extension](https://github.com/datastax/php-driver)

since the limitation is on datastax side, they will see that there's no `datastax/php-driver` compatible with `>= php7.2` ATM 🙏